### PR TITLE
SRVCOM-1727: Merge serverless-document-attributes into common-attributes

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+

https://issues.redhat.com/browse/SRVCOM-1727

Preview: -

### CODE REVIEW

File: cli_reference/kn-cli-tools.adoc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 :_content-type: ASSEMBLY
-include::_attributes/serverless-document-attributes.adoc[]
```
Comment (kalexand-rh) [inline cli_reference/kn-cli-tools.adoc:2]: You need to move the `common-attributes.adoc` include here or the attribute in your title won't render. (Ashleigh and I figured this out the hard way earlier.)

File: serverless/admin_guide/serverless-ossm-setup.adoc [hunk 2]
```diff
@@ -1,6 +1,4 @@
 :_content-type: ASSEMBLY
-include::_attributes/serverless-document-attributes.adoc[]
-include::_attributes/ossm-document-attributes.adoc[]
```
Comment (kalexand-rh) [inline serverless/admin_guide/serverless-ossm-setup.adoc:3]: This is going to cause your variables to break until Julie fixes the service mesh attribute files.

**Review by kalexand-rh — COMMENTED** [review]
This is looking good! I think it'll need to merge after the service mesh ones do, since you removed those attribute files, too. I suggest making the cherry-picks manually and confirming that each branch is correct. Please also squash.

File: serverless/admin_guide/serverless-ossm-setup.adoc (hunk 2 above)
Comment (rh-max) [reply serverless/admin_guide/serverless-ossm-setup.adoc:3]: ACK.

**Comment by kalexand-rh** [comment]
@rh-max, please squash. I'm happy to merge, but you're probably going to need to make manual CPs for 4.6 - 4.9.

**Pull request merged**